pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/juniperkits/JuniperKitUtil.java

```diff
@@ -0,0 +1,117 @@
+package org.broadinstitute.dsm.juniperkits;
+
+import static org.broadinstitute.ddp.db.TransactionWrapper.inTransaction;
+import static org.broadinstitute.dsm.service.admin.UserAdminService.PEPPER_ADMIN_ROLE;
+import static org.broadinstitute.dsm.statics.DBConstants.KIT_SHIPPING;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Arrays;
+import java.util.Collections;
+
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+import org.broadinstitute.dsm.kits.BaseKitTestUtil;
+import org.broadinstitute.lddp.db.SimpleResult;
+
+/**
+ * This class has methods to set up a study in DSM database that has the needed configuration to send out kits
+ * It creates the instance, group, instance_role, kit_type, carrier,
+ * kit_dimensions, kit_return and ddp_kit_request_settings for the
+ * newly created study.
+ * It also contains methods to delete what was set up after tests are complete
+ * </p>
+ * <p>
+ * The usage is by first creating an instance of this class by declaring the desired instance name and study guid,
+ * display name and collaborator prefix.
+ * Then call setupJuniperInstanceAndSettings() for initiating all the config in database.
+ *When done, call deleteJuniperInstanceAndSettings() to delete everything
+ * </p>
+ */
+@Slf4j
+@Getter
+public class JuniperKitUtil extends BaseKitTestUtil {
```

Comment:
This class has the config setup and what else that is only related to Juniper kits